### PR TITLE
Improvements in technology details page

### LINKED ISSUE #626: Melhorias no Layout do detalhamento da tecnologia
## Feature Description

Seguir esse layout atualizado:

Detalhes da Tecnologia - refatorado

- PREÇO: o preço agora é definido no campo "price"
- O botão ADQUIRIR ESSA TECNOLOGIA só fica disponível caso o campo "is_seller" for true.

- CUSTOS E FINANCIAMENTO
    - Retirar o bloco de custos de desenvolvimento.
    - Inserir abaixo dos custos o conteúdo do campo de observações de custos 

DOCUMENTOS
 - Diminuir o tamanho da thumb das imagens
 - Ao clicar na imagem aparecer um modal com a imagem em tamanho maior

RESPONSIVIDADE
- Quando visualizado em telas menores, as abas do detalhamento seguir a largura da tela, evitando tamanhos diferentes para cada aba.
---------------

## Acceptance criteria

* 

## Implementation Brief

*

### PR DESCRIPTION
**Pull request by luizeboli**
## Summary

Resolves #626 

https://www.loom.com/share/120c082fa0bc4809960c18eae24ed041

## QA Steps

1. Access any technology details page

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code passes the linting.
- [x] My code has proper inline documentation.
- [x] I have manually tested this PR.

### CODE REVIEW

File: packages/web/components/Technology/Details/Tabs/index.js [hunk 1]
```diff
@@ -21,31 +21,36 @@ const Tabs = () => {
...
+		</StyledTabs>
 	);
 };
 
+/**
+ * This isn't Next.JS getInitialProps
+ */
 Tabs.getInitialProps = () => {
 	resetIdCounter();
 };
```
Comment (mateus4k) [inline packages/web/components/Technology/Details/Tabs/index.js:47]: Maybe that's why it doesn't work. Can you put this code in the page component?

**Review by mateus4k — APPROVED** [review]
💯

**Review by mateus4k — APPROVED** [review]

File: packages/web/components/Technology/Details/Tabs/index.js (hunk 1 above)
Comment (luizeboli) [reply packages/web/components/Technology/Details/Tabs/index.js:47]: @mateus4k you call this method in some tests.  I couldnt understand your trakinagem

Comment (luizeboli) [reply packages/web/components/Technology/Details/Tabs/index.js:47]: Thanks for your contribution

**Review by calaca — APPROVED** [review]
All good! It looks so much better now. Congrats @luizeboli ! :clap:

**Pull request merged**